source/jormungandr/tests/line_sections_tests.py

```diff
@@ -69,6 +69,29 @@ def has_disruption(self, object_get, disruption_uri):
         r = self.default_query('{col}/{uri}'.format(col=object_get.collection, uri=object_get.uri))
         return has_disruption(r, object_get, disruption_uri)
 
+    def has_tf_disruption(self, q, disruption, pt_obj_type):
+        """
+        check a disruption is present in traffic report requests
+        check also that the disruption is linked to a certain object type
```

Comment:
this seems difficult for integrator that want to filter and print only disruptions affecting stop_point/area